lib/puppet/provider/cs_clone/crm.rb

```diff
@@ -81,38 +81,25 @@ def destroy
   # as stdin for the crm command.
   def flush
     unless @property_hash.empty?
-      if @property_hash[:existing_resource] == :false
-        debug 'Creating clone resource'
-        updated = 'clone '
-        updated << "#{@property_hash[:name]} "
-        updated << "#{@property_hash[:primitive]} "
-        meta = ''
-        meta << "clone-max=#{@property_hash[:clone_max]} " if @property_hash[:clone_max]
-        meta << "clone-node-max=#{@property_hash[:clone_node_max]} " if @property_hash[:clone_node_max]
-        meta << "notify=#{@property_hash[:notify_clones]} " if @property_hash[:notify_clones]
-        meta << "globally-unique=#{@property_hash[:globally_unique]} " if @property_hash[:globally_unique]
-        meta << "ordered=#{@property_hash[:ordered]} " if @property_hash[:ordered]
-        meta << "interleave=#{@property_hash[:interleave]}" if @property_hash[:interleave]
-        updated << 'meta ' << meta unless meta.empty?
-      else
-        debug 'Updating clone resource'
-        updated = 'resource meta '
-        updated << "#{@property_hash[:name]} "
-        meta = 'set '
-        meta << "clone-max=#{@property_hash[:clone_max]} " if @property_hash[:clone_max]
-        meta << "clone-node-max=#{@property_hash[:clone_node_max]} " if @property_hash[:clone_node_max]
-        meta << "notify=#{@property_hash[:notify_clones]} " if @property_hash[:notify_clones]
-        meta << "globally-unique=#{@property_hash[:globally_unique]} " if @property_hash[:globally_unique]
-        meta << "ordered=#{@property_hash[:ordered]} " if @property_hash[:ordered]
-        meta << "interleave=#{@property_hash[:interleave]}" if @property_hash[:interleave]
-        updated << meta unless meta.empty?
-        debug "Loading update: #{updated}"
-      end
+      updated = 'clone '
+      updated << "#{@resource.value(:name)} "
+      updated << "#{@resource.should(:primitive)} "
+      meta = ''
+      { clone_max: 'clone-max',
```

Comment:
our rubocop config normally marks this. What do you think about inserting a newline after {?